packages/connect-core/src/utils/path/calculatePath.ts

```diff
@@ -180,47 +150,44 @@ async function calculateForwardingPath(
  */
 export async function calculateTransactionPath(
   sender: string,
-  destination: string,
+  destinationApp: App,
   methodSignature: string,
   params: any[],
   apps: App[],
   provider: ethersProviders.Provider,
   finalForwarder?: string //Address of the final forwarder that can perfom the action. Needed for actions that aren't in the ACL but whose execution depends on other factors
-): Promise<PathData> {
-  // Get the destination app
-  const destinationApp = apps.find((app) => app.address == destination)
-  if (!destinationApp) {
-    throw new Error(
-      `Transaction path destination (${destination}) is not an installed app`
-    )
-  }
-
-  // Make sure the method signature is correct
-  const method = validateMethod(destination, methodSignature, destinationApp)
-
-  const finalForwarderProvided = finalForwarder
-    ? ethersUtils.isAddress(finalForwarder)
-    : false
+): Promise<Transaction[]> {
```

Comment:
The `calculateTransactionPath` just returns the list of Transaction objects now.